app/models/user.rb

```diff
@@ -1,4 +1,8 @@
+# Include default devise modules. Others available are:
 class User < ApplicationRecord
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] As part of this activity's requirements, we want a user to be asked for confirmation of email during the registration process, therefore let's ensure to enable the `:confirmable` option.